service-templates/apigw-lambda-svc/v1/instance_infrastructure/main.tf

```diff
@@ -1,117 +1,30 @@
-resource "aws_apigatewayv2_api" "lambda" {
-  name          = var.service.name
-  protocol_type = "HTTP"
-  cors_configuration {
-    allow_origins = ["*"]
-    allow_methods = [
-      "GET",
-      "HEAD",
-      "OPTIONS",
-      "POST",
-    ]
-  }
-
-  #  target = aws_lambda_function.lambda_function.arn
-}
-
-resource "aws_apigatewayv2_stage" "lambda" {
-  api_id = aws_apigatewayv2_api.lambda.id
+################################################################################
```

Comment:
`main.tf` in a module typically holds the actual source code for the resources (i.e. - https://github.com/bryantbiggs/terraform-aws-eks/blob/master/main.tf)

`main.tf` in a project though will generally hold generic information that is common throughout the project, including the remote backend settings, common local variables and data sources that are used throughout the project and not in any one specific resource or location